news.Rmd

```diff
@@ -2,12 +2,37 @@
 
 Each user-facing change to a package should be accompanied by a bullet in `NEWS.md`. Minor changes to documentation don't need to be documented, but it's worthwhile to draw attention to sweeping changes and to new vignettes.
 
-## Bullets
+## In-development
 
 The goal of the bullet is to briefly describe the change so users of the packages can understand what's changed. This can be similar to the commit message, but written with a user (not developer) in mind.
 
-New bullets should be added to the top of the file (under the first heading).
-Organisation of bullets will happen later, during the release process (Section \@ref(news-release)).
+New bullets should be added to the top of the file (immediatley under the first heading) and should be a single line (possibly wrapped). Organisation and wrapping will happen later, during the release process (Section \@ref(news-release)).
```

Comment:
```suggestion
New bullets should be added to the top of the file (immediately under the first heading) and should be a single line. Organisation and wrapping will happen later, during the release process (Section \@ref(news-release)).
```

You say `(possibly wrapped)` then say `wrapping will happen later`, I found that a little confusing. I think we just encourage a single line?

Reply:
Hmmm, I was trying to say that the line might be so long that it wraps around but I worded it in a very confusing way.